### DISCUSSION TITLE
Copilot is encountering temporary issues

### DISCUSSION THREAD

**Discussion by rubendr01** [community/community#14370] [Copilot Conversations] [answered]
Hi! I just install copilot extension on my laptop. I signed up and after verify my github account, the icon shows an alert message "Copilot is encountering temporary issues".

The output error message is: 
`[ERROR] [fetch] [2022-04-06T08:13:27.364Z] Unhandled status from server:,429,{"error":{"message":"Requests to the Create a completion from a chosen model Operation under OpenAI Language Model Instance API have exceeded rate limit of your current OpenAI S0 pricing tier. Please retry after 1 second. Please contact Azure support service if you would like to further increase the default rate limit."}}`

I love this extension and I want to use it.

Thank you!

**Comment by JACKROMER** [comment]
i have the same problem！ perhaps the ip has been limited.  or the github account's free test num used up.

**Reply by rubendr01** [reply]
I think it's only a visual warning, because I can work normally. I see this after post the issue

**Comment by itenl** [comment]
the issues happen to me, can't use.

**Comment by davecheney** [comment, marked as answer]
There has been an outage of one of our models which has caused the others to have to handle a higher traffic load, this is resulting in some temporary 429 responses. 

We’re working on a fix.

**Reply by RubenVerster** [reply]
Hey man. I'm not sure where is the best place to ask this, but do you know when a paid version of GHC will be coming out? <3

**Reply by ezzcodeezzlife** [reply]
still a problem right now on "text-davinci-002" model

**Reply by davecheney** [reply]
That isn't a model we offer. Could you please provide more details of what you did and what happened.

**Reply by ezzcodeezzlife** [reply]
im using the nodejs example from [the docs](https://beta.openai.com/docs/api-reference/completions/create). Inserted my API key. The request returns a 429 Status.

I tried text-davinci-002, text-curie-001, text-babbage-001, text-ada-001

 @davecheney

**Reply by davecheney** [reply]
I'm sorry, this is not the right place to ask for support for OpenAI's product.

**Comment by RubenVerster** [comment]
Guess we're copy-pasting from Google today gentlemen :)

**Reply by DevanHolmes** [reply]
I REFUSE TO COPY PASTE FROM GOOGLE, ILL STICK TO STACK OVERFLOW FOR NOW !!

**Reply by rubendr01** [reply]
I'm Spanish, my english is just so bad hahahahaha. I tried without copy paste from translate

**Reply by RubenVerster** [reply]
SO????

**Comment by RubenVerster** [comment]
Copilot seems to be working again. 
Thank you team! ^-^

**Comment by RubenVerster** [comment]
Looks like it's down again :')

I think that model is a bit tired of thinking today XP

**Comment by temrb** [comment]
issue still persists

**Reply by davecheney** [reply]
I'm sorry you are experiencing issues. I've checked your telemetry and cannot seen an 429's there so this is likely something unrelated. I'm going to close this issue and encourage you to create a new discussion entry. Please describe what you did, including your editor and version, and what happened when you tried. Thank you.
